components/hab/src/command/launcher.rs

```diff
@@ -22,7 +23,7 @@ const LAUNCH_PKG_IDENT: &str = "core/hab-launcher";
 
 pub async fn start(ui: &mut UI, sup_run: SupRun, args: &[OsString]) -> Result<()> {
     init()?;
-    let channel = sup_run.shared_load.channel;
+    let channel = SharedLoad::from(&sup_run).channel;
```

Comment:
I think this could just be `sup_run.channel` now and no need to transform to a `SharedLoad`.